tracing-attributes/tests/instrument.rs

```diff
@@ -43,8 +43,10 @@ fn override_everything() {
 
 #[test]
 fn fields() {
-    #[instrument(target = "my_target", level = "debug")]
-    fn my_fn(arg1: usize, arg2: bool) {}
+    struct UnDebug(pub u32);
+
+    #[instrument(target = "my_target", level = "debug", skip(_arg3, _arg4))]
```

Comment:
Take it or leave it, but: rather than adding the `skip` behavior to _this_ test, I might add a new test for the `skip` attribute? That way, if there's a regression that breaks only the `skip` attribute, we could see what behavior is broken based on which test fails. Not a blocker though!

Reply:
I'd be happy to add a new test, but I didn't do it mostly because I don't really understand how the span and subscriber mocking works, so adding into the existing test seemed easier.